pkg/plugins/shell/main_test.go

```diff
@@ -0,0 +1,53 @@
+package shell
+
+import "fmt"
+
+////////////// Test Utilities method aimed at mocking calls to other objects
```

Comment:
That's a whole lot of `//` 😁 😛

Reply:
I had to make sure that the reader see them :trollface:

Reply:
Well, is this comment really useful? Doesn't the typeName and the type comment already testifies of their purpose? :grin:

Reply:
Yep, I thought the same and removed it :)